draft-ietf-tls-tls13.md

```diff
@@ -3306,19 +3328,36 @@ ticket_lifetime
   for a shorter period of time than what is stated in the
   ticket_lifetime.
 
+ticket_extensions
+: A placeholder for extensions in the ticket. Clients MUST ignore
+  unrecognized extensions.
+
 ticket
 : The value of the ticket to be used as the PSK identifier.
-{:br }
-
 The ticket itself is an opaque label. It MAY either be a database
 lookup key or a self-encrypted and self-authenticated value. Section
 4 of {{RFC5077}} describes a recommended ticket construction mechanism.
+{:br }
 
+The meanings of the flags are as follows:
 
-[[TODO: Should we require that tickets be bound to the existing
-symmetric cipher suite. See the TODO above about early_data and
-PSK.??]
+allow_dhe_resumption
+: When resuming this session, the client MAY offer an (EC)DHE-PSK cipher
+  suite
```

Comment:
I think that this formulation isn't quite right.  It's not that the client MAY offer these cipher suites, because that is always possible.  It is "this ticket can be used with PSK cipher suites that have the following properties: PSK-only key exchange, PSK+DHE key exchange, (and later) PSK+signature authentication."